File: AMS_电子对账/code/server/source/com.yss.uco.elecreco/src/main/java/com/yss/uco/elecreco/er/reverse/map/assmap/service/IAssMapService.java
package com.yss.uco.elecreco.er.reverse.map.assmap.service;
import java.util.List;

import com.yss.uco.elecreco.er.reverse.map.assmap.pojo.AssMap;
import com.yss.framework.api.mvc.biz.IServiceBus;
import com.yss.framework.api.restful.annotations.RestfulSupported;

@RestfulSupported
public interface IAssMapService extends IServiceBus  {
	String getDzMode(String portCode,String fileType);
	List<AssMap> getCommonAssMapByPortCode(String portCode);
	List<AssMap> getAssMapByPortCodeAndFileType(String portCode,String fileType);
	List<String> getTghCodesByPortCode(String portCode);
}

File: AMS_电子对账/code/source/YssElecReco/Service/Er/Reverse/IAssMapService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using FAST.Common.Service.Services.Base;
using FAST.Common.Service.Attributes;
using YssElecReco.Context;

namespace YssElecReco.Service.Er.Reverse
{
    [ServiceAttribute(ServiceId = ElecrecoServiceIdConstant.OSGI_ELECRECO, ServiceController = "com.yss.uco.elecreco.er.reverse.map.assmap.controller.IAssMapServiceController")]
	public interface IAssMapService : IServiceBus  
	{
        [MethodAttribute]
        string getDzMode(string portCode, string fileType);
        [MethodAttribute]
        List<string> getTghCodesByPortCode(string portCode);
	}
}